### PR TITLE
Fix routing system

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h3>{{ site.title }}</h3>
       <ion-content>
         <ion-list *ngIf="!showSites">
           <ion-menu-toggle auto-hide="false" *ngFor="let p of appPages">
-            <ion-item lines="none" button [routerDirection]="'root'" [routerLink]="['/site', site.id, p.url]">
+            <ion-item lines="none" button [routerDirection]="'root'" [routerLink]="[p.url]">
```
Comment (Alir3z4) [inline src/app/app.component.html]: What's `p` and where does it get set ?

Comment (ArsalanSavand) [reply src/app/app.component.html]: The `p` is every object in appPages array.
Where they all have these structure:

```ts
  public appPages = [{
    title: 'Home', // A title
    url: '/home' // Page URL
    icon: 'home' // An ion icon
}];
```

Comment (Alir3z4) [reply src/app/app.component.html]: Oh I see, it's get set at `*ngFor="let p of appPages"`

name the variable `page` so it's cleaner.
At first I thought it's `param` :D

Comment (ArsalanSavand) [reply src/app/app.component.html]: Yea that makes sense.

Comment (ArsalanSavand) [reply src/app/app.component.html]: Done.

**Pull request merged**